WordPress/Classes/ViewRelated/Site Creation/Verticals/VerticalsWizardContent.swift

```diff
@@ -63,13 +69,30 @@ final class VerticalsWizardContent: UIViewController {
 
     private func setupTableBackground() {
         table.backgroundColor = WPStyleGuide.greyLighten30()
+        table.addTopBorder(withColor: WPStyleGuide.greyLighten20())
```

Comment:
Sylvester mentioned that the top & bottom strokes are missing. I think you're trying to accommodate that request here, but I'm not sure it's having the desired impact.

I believe this will add borders to the top & bottom of the gray region of the table view. In `SiteInformationWizardContent`, you added these borders to the top & bottom **rows** (i.e., cells) in the table view.